src/lib/services/types/proposal.ts

```diff
@@ -363,15 +230,56 @@ const zProposalAnswerCountsResponse = z
     }),
   })
   .transform(snakeToCamel);
-
 export type ProposalAnswerCountsResponse = z.infer<
   typeof zProposalAnswerCountsResponse
 >;
 
-export const getProposalAnswerCounts = async (
-  endpoint: string,
-  id: number
-): Promise<ProposalAnswerCountsResponse> =>
-  axios
-    .get(`${endpoint}/${encodeURIComponent(id)}/answer-counts`)
-    .then(({ data }) => parseWithError(zProposalAnswerCountsResponse, data));
+export const zProposalsResponseItemLcd = z
+  .object({
+    id: z.coerce.number(),
+    messages: z
+      .object({
+        "@type": z.string(),
+      })
+      .passthrough()
+      .array()
+      .nullable(),
```

Comment:
@songwongtp I think we can remove nullable, right?

Reply:
yep cannot find any proposal with messages as `null`